ci-operator/config/maistra/envoy/maistra-envoy-maistra-2.2.yaml

```diff
@@ -78,6 +78,22 @@ tests:
       timeout: 8h0m0s
     workflow: servicemesh-envoy-e2e-aws
   timeout: 8h0m0s
+- as: envoy-update-proxy-2.2
+  commands: |
+    ./tools/automator-main.sh \
```

Comment:
where is the tools/automator-main.sh ?
https://github.com/maistra/envoy/tree/maistra-2.4/tools

This job container from: src means it starts from https://github.com/maistra/envoy